kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -320,6 +320,39 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
           appStateSource,
           appStateContainer)
       }
+
+      try {
+        val volumes = pod.getSpec.getVolumes.asScala
```

Comment:
It might take time to make this action, could you use executor service/thread pool?

Reply:
<img width="948" alt="image" src="https://github.com/user-attachments/assets/791bbb77-5482-42fd-8525-aa33670f7c7f">

Reply:
Hi @turboFei , I have implemented it using ThreadPool, could you please review. Was it the expectation?